src/libs/NumberCovert.js

```diff
@@ -0,0 +1,57 @@
+import lodashGet from 'lodash/get';
+import loadashTrim from 'lodash/trim';
+import loadashTrimStart from 'lodash/trimStart';
+import loadashIncludes from 'lodash/includes';
+import loadashStartsWith from 'lodash/startsWith';
+import loadashPadStart from 'lodash/padStart';
+import translations from '../languages/translations';
+
+/**
+ * Returns a locally converted phone number without the country code
+ *
+ * @param {String} locale eg 'en', 'es-ES'
+ * @param {String} number
+ * @returns {string}
+ */
+function toLocalPhone(locale, number) {
+    const numString = loadashTrim(number);
```

Comment:
Hmmmm is `trim` and its variants what we want here and really in the rest of this file? I think not, since `trim` removes all passed characters from the beginning/end of the string. So if you do `trim(`343443344`, '34')` you will get back an empty string. Right? If so, please add a test case that fails before fixing this.

Reply:
Here trim is just being used to remove spaces from the start and end of the number. I have added a test case with spaces at the beginning and end of number already. 

For example expect(numConvert.fromLocalPhone('en', ' + 34 747 474 7474 ')).toBe('+1 34 747 474 7474');

Reply:
Sorry, I put this comment in the wrong line. I meant to put it [here](https://github.com/Expensify/Expensify.cash/pull/2130/files#diff-bb1f95965dba63a26b72d87f27c9c0e3a93ceceda725b41ae533059f2f75fb1eR23).

Reply:
Very good point raised. I have written this test that failed with our usage of trimStart method from loadash.

// Failing due to the use of Trimstart.
    it('Test to local Number Conversion by locale', () => {
        expect(localePhoneNumber.toLocalPhone('es-ES', '343434343434')).toBe('3434343434');
    });

I have rectified this issue by using Str.cutBefore from our existing utilities. I hope it is fine now.